src/api/Metadata.js

```diff
@@ -257,6 +267,64 @@ class Metadata extends File {
         }
     }
 
+    /**
+     * Gets classification for a file.
+     *
+     * @param {BoxItem} file - File object for which we are getting classification
+     * @param {Function} successCallback - Success callback
+     * @param {Function} errorCallback - Error callback
+     * @param {boolean|void} [options.forceFetch] - Optionally Bypasses the cache
+     * @param {boolean|void} [options.refreshCache] - Optionally Updates the cache
+     * @return {Promise}
+     */
+    async getClassification(
+        file: BoxItem,
+        successCallback: Function,
+        errorCallback: Function,
+        options: FetchOptions = {},
+    ): Promise<void> {
+        const { id }: BoxItem = file;
+        if (!id) {
+            errorCallback(getBadItemError());
+            return;
+        }
+
+        const cache: APICache = this.getCache();
+        const key = this.getClassificationCacheKey(id);
+        this.successCallback = successCallback;
+        this.errorCallback = errorCallback;
+
+        // Clear the cache if needed
+        if (options.forceFetch) {
+            cache.unset(key);
```

Comment:
why unset if its a forceFetch? Maybe just `if (cache.has(key) && !options.forceFetch)`

Reply:
This is how we use `forceFetch` in multiple places. Following suit, here. Leaving as is

Reply:
unset is here, because the line below checks if its set below